core/build.gradle.kts

```diff
@@ -408,6 +441,11 @@ kotlinPublications {
         description = "Dataframe core API"
         packageName = artifactId
     }
+    publication {
+        publicationName = "dataframe-experimental"
+        description = "Excel support for Kotlin Dataframe"
+        composeOfTaskOutputs(listOf(experimentalJar, experimentalJarSources))
```

Comment:
omg, I'm gonna have to look into doing this for the publishing of generated sources too. That's way easier than temporarily modifying the kotlin main sourceset before publishing each time.